DEPLOY.md

```diff
@@ -35,6 +35,21 @@ You'll need to create an AWS account, if you don't already have one. Then, you'l
 }
 ```
 
+For the officer identification UI to work, you'll need to create a CORS policy for the S3 bucket used with OpenOversight. Since we're not doing anything fancier than making a web browser GET it, we can just use the default policy:
```

Comment:
For someone who has never set a CORS policy in S3, it is unclear where this seeing would be set. Needs a bit more documentation.